### ISSUE TITLE
`templates/icons/*.svg` not included in release package

### ISSUE THREAD

**Issue by mixxorz** [wagtail-nest/wagtail-ab-testing#88]
I tried installing 0.11 and found that I get this error:

```
  File "/venv/lib/python3.11/site-packages/django/template/loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: wagtail_ab_testing/icons/crown.svg
```

Upon further inspection, it looks like the release uploaded to PyPI does not contain these files. So something's wrong with the release process that the `*.svg` files are not included in the final package.

**Comment by katdom13** [comment]
@mixxorz , please see this PR:
https://github.com/wagtail-nest/wagtail-ab-testing/pull/89

Thanks

**Comment by Stormheg** [comment]
@mixxorz thanks for reporting! This is now fixed in v0.11.1
